macros/tests/ui/gservice_fails_multiple_not_allowed_on_one_impl.stderr

```diff
@@ -0,0 +1,14 @@
+error: multiple `gservice` attributes are not allowed
+ --> tests/ui/gservice_fails_multiple_not_allowed_on_one_impl.rs:7:1
+  |
+7 | impl MyService {
+  | ^^^^
+
+error[E0599]: no method named `this` found for struct `MyService` in the current scope
```

Comment:
is it the actual error message that user will see?

if yes, can we remove further errors because macros is invalid on the first place?

Reply:
Wondered as well why this message pops out there

Reply:
that's the second error, trying to figure how to remove it

Reply:
Weird, it should stop as soon as `abort!` is called. Would be nice if we can get rid of this second message

Reply:
I think the first attribute has already generated some code, then the second call `abort!`

Reply:
added new method  `ensure_single_gservice_on_impl`